gxf_extensions/deltacast_videomaster/videomaster_base.hpp

```diff
@@ -62,13 +65,13 @@ class VideoMasterBase : public gxf::Codelet {
 
   bool signal_present();
   bool set_loopback_state(bool state);
-  std::unordered_map<ULONG, ULONG> get_detected_input_information(uint32_t channel_index);
-  std::unordered_map<ULONG, ULONG> get_input_information();
 
-  bool api_call_success(ULONG api_error_code, std::string error_message);
```

Comment:
Wouldn't it still be useful?
The code has become a tad bit more verbose with all the ifs after each api call.